Packs/Cylance_Protect/Integrations/Cylance_Protect_v2/Cylance_Protect_v2.py

```diff
@@ -40,6 +41,9 @@
 SCOPE_THREAT_LIST = 'threat:list'
 SCOPE_GLOBAL_LIST_CREATE = 'globallist:create'
 SCOPE_GLOBAL_LIST_DELETE = 'globallist:delete'
+SCOPE_OPTICS_LIST = 'opticssurvey:list'  # Get InstaQueries
+SCOPE_OPTICS_CREATE = 'opticssurvey:create'  # Create InstaQuery
+SCOPE_OPTICS_GET = 'opticssurvey:read'  # Read a InstaQuery
```

Comment:
Where are these used?

Reply:
Added these scope to the get_authentication_token() in each command.